packages/wonder-blocks-form/components/radio-group.js

```diff
@@ -89,9 +89,11 @@ export default class RadioGroup extends React.Component<RadioGroupProps> {
             <StyledFieldset style={styles.fieldset}>
                 {/* We have a View here because fieldset cannot be used with flexbox*/}
                 <View style={style}>
-                    <legend>
-                        <LabelMedium>{label}</LabelMedium>
-                    </legend>
+                    {label && (
+                        <legend style={{padding: 0}}>
```

Comment:
🛑 Same here with StyleSheet.

Reply:
@wishdasher did this get addressed in a followup PR?

Reply:
It was addressed in this one!

Reply:
My bad, usually GitHub hides comments on lines that have changed.